spec/paychex/client/companies_spec.rb

```diff
@@ -1,16 +1,20 @@
 RSpec.describe 'Paychex' do
   describe 'companies' do
     it 'linked_companies should return list' do
-      stub_get('companies').to_return(
-        body: fixture('companies/companies.json'),
-        headers: { content_type: 'application/json; charset=utf-8' }
-      )
+      stub_get('companies?limit=100&offset=0').
```

Comment:
Would you please add another test case to cover `multiple pages`?

Reply:
Updated the test case since the `linked_companies` will now return paginated response only.